how-it-works/00_General/01_overview-of-the-internet-computer.card.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Architecture of the Internet Computer
+---
+
+![](/img/how-it-works/subnet_architecture.png)
+
+# Architecture of the Internet Computer
+
+The Internet Computer implements the vision of a World Computer –  an open and secure blockchain-based network that can host all of humanity's computations in a secure and trustworthy way, witout needing to trust any single party to behave properly.
+To achieve this, the technology behind the Internet Computer (IC) had to be reconceived from ground up, learning many lessons from earlier projects that failed to create a World Computer.
+
+At a high level, the IC's architecture builds on *subnets*, or *subnetworks*: The IC as a whole consists of many subnets, where each subnet is its own blockchain that operates concurrently with and independently of the other subnets.
```

Comment:
Why do we start with subnets? To me, subnets are the answer to the question: "How does the IC scale?" - but that would be something worth covering after the main functional description (i.e.: run canisters in a replicated, fault-tolerant way).